### ISSUE TITLE
Unable to work with new version - Typeorm not working

### ISSUE THREAD

**Issue by panbhatt** [TypedProject/tsed-example-react#2]
Hi Team,
 this boilerplate seems not be working after we integrated TYPEORM in the project. Let me know, if i can do a pull request or create a new repository with React/TypeORM.

**Comment by Romakita** [comment]
Yes you can create Pull request :)

**Comment by Romakita** [comment]
Originally this repository is just here to showing React with Ts.ED. Not to show TypeORM ^^ (there is a specific repo for that). 
With your PR, we introduce TypeORM but the database isn't used by the backend code. 
So your goal is to implement something to use it? 

See you ;)

**Comment by panbhatt** [comment]
It's being used. If we load the react front page a console message is displayed in the server logs, telling the no of employees present in the database. The code is present in 62-65 line in CalendarServive ts

**Comment by panbhatt** [comment]
So sorry that I had integrate it here. If you don't want it here we can create another icon on tsed.io mentioning react n typeorm integration.

**Comment by Romakita** [comment]
No worries @panbhatt ^^

**Comment by panbhatt** [comment]
Thanks. Originally I thought it would be good to integrate typeorm with react as react boilerplate is too old and not working with typeorm tsed new release version. Anywhere I can help would be a pleasure.

**Comment by Romakita** [comment]
Yep you've right :) and it's good idea. I think the example should be really better if it use only the database instead of mocked data with MemoryService. I totally missed the line where the employee number is checked ^^. 
So if you have a time to improve the example you can replace all methods that use MemoryService and use TypeORM :)
It'll be a real pleasure to review it and help you on it ;)

On the TypeORM example I added an integration test example. I hope this example will help you^^

**Comment by panbhatt** [comment]
Sure. I can do one react typeorm with sqlite demo replacing memory service with entity repository. And take typeorm integration test case example from other repo. Question will it be a new repo or pull request to another repo.

**Comment by Romakita** [comment]
Keep on this repo ;)
